### PR TITLE
Ashley/landing page

### PR DESCRIPTION
**Pull request by ashleydvss**

### CODE REVIEW

**Comment by ashleydvss** [comment]
Test

File: cat/package-lock.json [hunk 1]
```diff
@@ -2921,10 +3002,31 @@
         "url": "https://github.com/sponsors/isaacs"
       }
     },
+<<<<<<< HEAD
```
Comment (ComTDK) [inline cat/package-lock.json]: LMAO, resolve this bro

File: cat/src/app/page.js [hunk 2]
```diff
@@ -1,32 +1,92 @@
...
 
   return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
-      Pet app: {user ? user.email : "Dume"}
-      <AddNotificationForm />
-      <LogOutButton />
+    <div className="relative z-1">
+      <Background />
+      <div className="relative z-2 flex flex-col items-center justify-center min-h-screen">
+        <h2 className="text-center">meow!</h2>
```
Comment (ComTDK) [inline cat/src/app/page.js:73]: Cannot see this
image

image

File: cat/package-lock.json (hunk 1 above)
Comment (ashleydvss) [reply cat/package-lock.json]: done done

**Review by ComTDK — APPROVED** [review]

**Pull request merged**